docs/Typography.md

```diff
@@ -0,0 +1,65 @@
+# Typography
+
+## When using the library  
+
+`wix-ui-tpa` components are wired to the site's fonts as predefined in the [design system](https://zeroheight.com/7sjjzhgo2/p/7181b5-tpa-design-system).  
+In most cases, no extra work is needed in order to ensure the components render with the correct font-family, size and color.  
+
+
+### Titles and body text:
+The recommended way to use `wix-ui-tpa` typography is with the `<Text/>` component.
+
+The `<Text/>` component receives a `typography` prop, which its values are mapped to the font according to the following table:
+
+| Typography  | Font theme | Size | Line Height |
+| ----------  | -----      | ----- | ----- |
+| smallTitle  | Page-title | Desktop: 24px <br /> Mobile: 20px | Desktop: 1.33 <br /> Mobile: 1.4 |
+| largeTitle  | Heading-M  | Desktop: 32px <br /> Mobile: 24px | Desktop: 1.25 <br /> Mobile: 1.33 |
+| listText    | Body-M     | Desktop: 16px <br /> Mobile: 14px | Desktop: 2 <br /> Mobile: 1.72 |
+| runningText | Body-M     | Desktop: 16px <br /> Mobile: 14px | Desktop: 1.5 <br /> Mobile: 1.42 |
+
+It is rendered as a `span` by default, but this can be changed using the `tagName` prop
+
+#### Example
+
+```jsx
+import { Text, TYPOGRAPHY } from 'wix-ui-tpa/Text';
+...
+export const MyApp = () => {
+  return (
+    <section>
+      <Text tagName={'h2'} typography={TYPOGRAPHY.largeTitle}>Hello World!</Text>
+      <Text typography={TYPOGRAPHY.runningText}>The quick brown fox jumps over the lazy dog</Text>
+    </section>  
+  );
+}
+```
+
+
+## When contributing to the library
+When creating a new component that renders text, we should in most cases use the `<Text/>` component with the correct props.  
+In some cases we might need to use a typography that isn't defined in the `<Text/>` component.  
+In this case, we rely on the style processor's syntax, to ensure we get the correct font from the viewer.
```

Comment:
everything that is contribution related should go to a contribution file(s).
As a consumer of a library, I'd like to focus on how I consume it and not how to work on it.

Reply:
The combination of your comments effectively suggest simply removing this document 😄 
What do you think should appear in this document?

Reply:
Yes, kind of.
I think that having a well structured `Text` component story page that includes also the [ZeroHeight Specs](https://zeroheight.com/7sjjzhgo2/p/00b9e9-typography) and a section about the style processor mapping to the design name will be enough

Reply:
The problem at the moment is that the `Text` component is a bit out-of-date, so we can't rely on it too much.
I would like to update it soon.
But until then, I think we need a general guide.

I can move the contribution content to the contribution guide, and remove this doc.
Although I think it's also good for consumers to know how this works.

Reply:
ok